programs/github-pull-request/getGithubAppUserToken/index.js

```diff
@@ -0,0 +1,18 @@
+const GitHubApp = require('../../../lib/GithubApp');
+
+/**
+ * Get a github app installation token.
+ * @param  {String} options.appId
+ * @param  {String} options.appPrivateKey
+ * @return {Promise<String | undefined>}
+ */
+module.exports = async function getGithubAppUserToken({appId, appPrivateKey}) {
+	if (!(appId && appPrivateKey)) {
+		return;
+	}
+
+    return new GitHubApp({
+        appId,
+        appPrivateKey: Buffer.from(appPrivateKey, 'base64').toString(),
```

Comment:
I would move the `Buffer.from` to the constructor of the class, to encapsulate as much from the "caller".